osf/models/preprint.py

```diff
@@ -921,8 +1166,12 @@ def bulk_update_search(cls, preprints, index=None):
             logger.exception(e)
             log_exception(e)
 
-    def update_search(self):
-        update_share(self)
+    def update_search(self, skip_share=False):
+        """Update SHARE and OSF search.
+        """
+        if not skip_share:
```

Comment:
Minor: It seems like all of the `skip_share` changes could've been avoided if this conditional checked `self.is_latest_version` instead, unless there's a use case I'm missing.

Reply:
Fixed in https://github.com/CenterForOpenScience/osf.io/pull/10946